me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -50,11 +51,11 @@
       <classpathProvider
           id="me.gladwell.m2e.android.sourcepathProvider"
           class="me.gladwell.eclipse.m2e.inject.GuiceExtensionFactory:me.gladwell.eclipse.m2e.android.JUnitSourcepathProvider" />
-   </extension>
-   <extension
-         point="org.eclipse.m2e.core.mavenProjectChangedListeners">
-      <listener
-            class="me.gladwell.eclipse.m2e.inject.GuiceExtensionFactory:me.gladwell.eclipse.m2e.android.EclipseEventsToEventBusListener">
-      </listener>
-   </extension>
+   </extension>
```

Comment:
@rgladwell these lines had CRLF line endings, so git changed them to LF when i committed it. You should add [this](https://help.github.com/articles/dealing-with-line-endings/#global-settings-for-line-endings) config to your Windows machine.